tests/test_cl_half.h.c

```diff
@@ -95,7 +95,7 @@ int main(void)
   CHECK_FROM_FLOAT(-(CL_HALF_MAX + 1000.f), 0xfbff, RTP);
   CHECK_FROM_FLOAT(-(CL_HALF_MAX + 1000.f), 0xfc00, RTN);
   CHECK_FROM_FLOAT(-(CL_HALF_MAX + 1000.f), 0xfbff, RTZ);
-
+#ifndef _MSC_VER
```

Comment:
How are we dealing with this in the CTS? Maybe we could wrap hex float constants in macros and make this transparent (just thinking out loud).

Reply:
I don't see host-side tests for cl_half, only those which load-store halfs from various memory spaces and the combination, doing a roundtrip to device and back. There are no rounding tests. These on MSVC are hard, syntax errors. Opened #133 to defer resolution.

Reply:
Fine with deferring to a follow on issue.